content/sensu-core/1.2/guides/snmp-sensu-guide.md

```diff
@@ -0,0 +1,252 @@
+---
+title: "How to Monitor Devices With Sensu's SNMP Receiver Integration"
+product: "Sensu Enterprise"
+version: "2.8"
+weight: 1
+menu:
+ sensu-enterprise-2.8:
+   parent: guides
+---
+
+# Objectives
+- Set up a Sensu client as an SNMP trap receiver
+- Send a test SNMP trap to simulate a real world circumstance
+- Demonstrate the results of the SNMP trap test in Sensu Enterprise Dashboard
+
+_NOTE: This guide applies only to Sensu Enterprise._ 
```

Comment:
Let's remove/reword these mentions of Sensu Enterprise, as this guide applies to any combination of sensu-client + event processor and dashboard.